packages/block-editor/src/components/block-tools/index.js

```diff
@@ -152,6 +153,12 @@ export default function BlockTools( {
 				// block so that focus is directed back to the beginning of the selection.
 				// In effect, to the user this feels like deselecting the multi-selection.
 				selectBlock( clientIds[ 0 ] );
+			} else if ( clientIds.length === 1 ) {
+				event.preventDefault();
+				clearSelectedBlock();
+				__unstableContentRef?.current
+					?.querySelector( '[aria-label="Block canvas"]' )
```

Comment:
What is `[aria-label="Block canvas"]`? I can't find such element in the DOM. What about focusing `__unstableContentRef`?

Reply:
I think this was a bug due to rebasing. It should be sending focus to the region, as that seems to be the best one for screen reader announcements.